web/src/pages/Login/Login.tsx

```diff
@@ -49,7 +49,7 @@ export default function Login() {
             try {
                 await axios
                     .post(
-                        `${config.apiUrl}/login`,
+                        `http://localhost:8080/login`,
```

Comment:
this URL will not work in production

Reply:
i can look into it tomorrow no worries. can you try to fix the github actions in the meantime ?